archive/imo1988_q6.lean

```diff
@@ -0,0 +1,271 @@
+import data.rat.basic
```

Comment:
Files in the archive should have copyright headers, too.

Reply:
Done